### ISSUE TITLE
Select default capitalizations and styles based on dialect

### ISSUE THREAD

**Issue by BrainStone** [jOOQ/jOOQ#9495]
### Use case:

So the idea is simple, many dialects have their own defaults and standards for capitalization and other styles (like quoting) which they use throught their documentation.  
Now it would pretty nice if there were an additional option in the dropdowns that indicates it is going to use that dialects default choice.

### Possible solution you'd like to see:

Ideally an additional option in the dropdowns that use the dialects defaults.  
Alternatively changing the dialect will just preselect its defaults, though it might be upsetting because it might change your own settings.

**Comment by lukaseder** [comment]
Thanks a lot for your suggestion. To be sure, this is about our SQL translation website here, right? https://www.jooq.org/translate

In that case, I agree that such an option would be cool. We could solve it with a "use default" checkbox column. As soon as users change the dropdown value, the checkbox would uncheck, and the user choice would be sticky. Checking it again would select the dropdown value that corresponds to the current dialect's default.

**Comment by BrainStone** [comment]
Yes I'm talking about the SQL translator. When opening the issue I though that was all this repo was for, which I now know is not the case.

Anyways I agree that your suggestion would work too.

**Comment by BrainStone** [comment]
Also for the CLI tool (and the API) if no value is specified it uses that default too.

**Comment by lukaseder** [comment]
Yes, of course. This could be implemented as a jOOQ library feature, and then exposed to all the clients, including website, CLI, etc.
